packages/migration/src/migrations/v0002_dataPixelTo1080.ts

```diff
@@ -22,41 +28,51 @@ const OLD_PAGE_HEIGHT = 732;
 const SCALE_X = NEW_PAGE_WIDTH / OLD_PAGE_WIDTH;
 const SCALE_Y = NEW_PAGE_HEIGHT / OLD_PAGE_HEIGHT;
 
-function dataPixelTo1080({ pages, ...rest }) {
+export type StoryV2 = StoryV1;
+export type PageV2 = PageV0;
+export type UnionElementV2 = UnionElementV0;
```

Comment:
This makes it easier, because you always just extend the definition from the previous migration, but you'd have to do so for all the types. not just these. Also for each separate element type and potentially also all the resource types, etc.